public/js/lib/devtools-sham/client/shared/key-shortcuts.js

```diff
@@ -141,7 +141,7 @@ KeyShortcuts.parseElectronKey = function (window, str) {
     key = ElectronKeysMapping[key];
     shortcut.keyCode = window.KeyboardEvent[key];
     // Used only to stringify the shortcut
-    shortcut.keyCodeString = key;
+    shortcut.key = key;
```

Comment:
@jlongster, tomorrow is Jaideep's last day. I'd be happy to file an m-c PR next week after this lands.